app/views/patients/unknown_patient.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-24 d-flex flex-column justify-content-center">
+      <%= render partial: 'errors/logo' %>
+      <div class="row">
+        <div class="mx-auto">
+          <h2>Monitoree Unavailable</h2>
```

Comment:
This still doesn't look like the mockups.  Perhaps it would be easier to make this a React component for styling purposes?